index.bs

```diff
@@ -3316,13 +3316,20 @@ Maturity Stages on the Recommendation Track</h4>
 			and that no further refinement to the text is expected
 			without additional implementation experience and testing;
 			however, additional features might be expected in a later revision.
-			A [=Candidate Recommendation=] is expected to be as well-written,
+			The word <em>Candidate</em> in [=Candidate Recommendation=] signals
+			that the document is <em>eligible</em> for evaluation against the
+			requirements for further advancement — but does not indicate that
+			the Working Group is necessarily <em>intending</em> further advancement,
+			nor that the Working Group necessarily believes the document already meets
+			the requirements for further advancement. But a [=Candidate Recommendation=]
```

Comment:
Refining the phrasing of @nigelmegitt's suggestion, needed another two lines to be selected as suggestion input...
```suggestion
			requirements for further advancement — but does not indicate whether
			the Working Group believes the document already meets
			those requirements. Nonetheless, a [=Candidate Recommendation=]
```